### DISCUSSION TITLE
consolidate tags into FAQs CT

### DISCUSSION THREAD

**Discussion by ComfortablyCoding** [strapi-community/strapi-ai-bot#13] [General]
Having both tags and FAQs as seperate cts is basically redundant. The code and commands are fairly similar and their is not a lot of difference between the two.

We have two potential solutions
1. We remove tag CT and keep FAQ CT as is
2. We remove tag CT and add a tags relation for FAQ that acts as keywords in FAQ search.

**Comment by derrickmehaffy** [comment]
Vote for option 1
